Tools/CISupport/ews-build/steps.py

```diff
@@ -1249,13 +1249,13 @@ class CheckChangeRelevance(AnalyzeChange):
     ]
 
     jsc_path_regexes = [
-        re.compile(rb'.*jsc.*', re.IGNORECASE),
         re.compile(rb'.*javascriptcore.*', re.IGNORECASE),
         re.compile(rb'JSTests/', re.IGNORECASE),
         re.compile(rb'Source/WTF/', re.IGNORECASE),
         re.compile(rb'Source/bmalloc/', re.IGNORECASE),
         re.compile(rb'Source/cmake/', re.IGNORECASE),
         re.compile(rb'.*Makefile.*', re.IGNORECASE),
+        re.compile(rb'Tools/.*jsc.*', re.IGNORECASE),
```

Comment:
Nit: This could be `Tools/Scripts/.*jsc.*` to be clear about where the possible JSC scripts live.

Reply:
I went with `Tools` because I figured it's possible some JSC related things could live outside `Tools/Scripts` and this would catch anything inside `Scripts`